client/src/components/Trivalry/HostPage/index.tsx

```diff
@@ -128,7 +128,7 @@ export function TrivalryHostPage() {
                                             <Paper elevation={3} sx={{py: 2, px: 2}} style={{background: "#E3CAA5", height: "350px"}}>
                                                 <Typography color="error">Room Code</Typography>
                                                 <Box sx={{mx:1, my:14}}>
-                                                    <Typography>{gameStats.gameCode}</Typography>
+                                                    <Typography fontSize = "32">{gameStats.gameCode}</Typography>
```

Comment:
@fkery this syntax does not actually change the font size... replace the "" with {} and then you will see the change you are suggesting! The line below should do the trick!

`<Typography fontSize={32} >{gameStats.gameCode}</Typography>`

Reply:
Just made the changes. Let me know if there's anything else I should do.